### PR TITLE
Update 03-sql-joins.md

### PR DESCRIPTION
**Pull request by cjcenter**
I have inserted a preamble to the section discussing joins relating the concept to relational databases and associated tabular structure.

### CODE REVIEW

File: _episodes/03-sql-joins.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ keypoints:
 To combine data from two tables we use the SQL `JOIN` command, which comes after
 the `FROM` command.
 
+Earlier, we talked about relational databases.
```
Comment (remram44) [inline _episodes/03-sql-joins.md]: I think I would remove this, it's not clear what the relation is with the next paragraph (are you hinting at normal forms?)

File: _episodes/03-sql-joins.md [hunk 2]
```diff
@@ -21,6 +21,17 @@ keypoints:
...
+
+Database tables are used to organize and group data by common characteristics or principles.  
+Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
+A JOIN is a means for combining columns from one (self-join) or more tables by using values common to each.
```
Comment (remram44) [inline _episodes/03-sql-joins.md]: I would leave out self-joins. They are not that much of a special case (you can use a single table multiple times even in the multiple-table case).

File: _episodes/03-sql-joins.md [hunk 3]
```diff
@@ -21,6 +21,17 @@ keypoints:
...
+
+The JOIN command combined with ON is used to combine fields from separate tables.  
+
+An SQL join clause combines columns from one or more tables in a relational database. 
+It creates a set that can be saved as a table.
```
Comment (remram44) [inline _episodes/03-sql-joins.md]: Saving the results of the query as a new table is not that common and goes against normalization. Or do you mean a file, e.g. CSV?

File: _episodes/03-sql-joins.md (hunk 1 above)
Comment (ChristinaLK) [inline _episodes/03-sql-joins.md:24]:
```suggestion

```

File: _episodes/03-sql-joins.md (hunk 2 above)
Comment (ChristinaLK) [inline _episodes/03-sql-joins.md:28]:
```suggestion
A JOIN is a means for combining columns from multiple tables by using values common to each.
```

File: _episodes/03-sql-joins.md [hunk 4]
```diff
@@ -21,6 +21,16 @@ keypoints:
...
 the `FROM` command.
 
+
+Database tables are used to organize and group data by common characteristics or principles.  
+Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
+A JOIN is a means for combining columns from multiple tables by using values common to each.
+
+The JOIN command combined with ON is used to combine fields from separate tables.  
+
+An SQL join clause combines columns from one or more tables in a relational database.
```
Comment (ChristinaLK) [inline _episodes/03-sql-joins.md:31]:
```suggestion

```

File: _episodes/03-sql-joins.md [hunk 5]
```diff
@@ -21,6 +21,16 @@ keypoints:
...
 
+
+Database tables are used to organize and group data by common characteristics or principles.  
+Often, we need to combine elements from separate tables into a single tables or queries for analysis and visualization.
+A JOIN is a means for combining columns from multiple tables by using values common to each.
+
+The JOIN command combined with ON is used to combine fields from separate tables.  
+
+An SQL join clause combines columns from one or more tables in a relational database. 
+It creates a set that can be saved as a table.
```
Comment (ChristinaLK) [inline _episodes/03-sql-joins.md:32]:
```suggestion

```

**Pull request merged**